src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,82 @@
+import classNames from 'classnames';
+import React from 'react';
+
+interface Props {
+  todosCount: number;
+  completedCount: number;
+  filter: 'all' | 'active' | 'completed';
+  setFilter: (filter: 'all' | 'active' | 'completed') => void;
+}
+
+export const Footer: React.FC<Props> = ({
+  todosCount,
+  completedCount,
+  filter,
+  setFilter,
+}) => {
+  const handleFilterChange = (newFilter: 'all' | 'active' | 'completed') => {
+    setFilter(newFilter);
+  };
+
+  const getFilteredTodosCount = () => {
+    switch (filter) {
+      case 'all':
+        return todosCount;
+      case 'active':
+        return todosCount - completedCount;
+      case 'completed':
+        return completedCount;
+      default:
+        return completedCount;
+    }
+  };
+
+  const filteredTodosCount = getFilteredTodosCount();
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${filteredTodosCount} ${filteredTodosCount === 1 ? 'item' : 'items'} left`}
+      </span>
+
+      <nav className="filter">
+        <a
+          href="#/"
+          className={classNames('filter__link', {
+            selected: filter === 'all',
+          })}
+          onClick={() => handleFilterChange('all')}
+        >
+          All
+        </a>
+        <a
+          href="#/active"
+          className={classNames('filter__link', {
+            selected: filter === 'all',
+          })}
+          onClick={() => handleFilterChange('active')}
+        >
+          Active
+        </a>
+        <a
+          href="#/completed"
+          className={classNames('filter__link', {
+            selected: filter === 'all',
+          })}
+          onClick={() => handleFilterChange('completed')}
+        >
+          Completed
+        </a>
+      </nav>
+
+      {completedCount > 0 && (
```

Comment:
let's add a class with `visibility: hidden` style instead of conditional rendering to keep the filter in the center of the footer: 
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/86923089/bc759e91-73f7-45a3-b3aa-c4af8a5887a4)